### DISCUSSION TITLE
overriding materialization from custom dbt_package

### DISCUSSION THREAD

**Discussion by ozerpekozcan** [dbt-msft/dbt-sqlserver#546] [Q&A]
Hello

I would like to know if it's possible to override materializations from custom dbt package derived from dbt-sqlserver package. 
It is possible to override if the materialization macro inside the project. However I would like to do it inside my custom package where other customized macros are in.

**Comment by cody-scott** [comment]
Sounds like a question that may be better suited to the dbt slack channel; you'll get a wider response there. 

I haven't done it myself, but you could check out this doc and see if it helps. 

https://docs.getdbt.com/guides/create-new-materializations?step=1

This would be helpful to understand as well. 

https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch
